.eslintrc.js

```diff
@@ -1,5 +1,12 @@
 module.exports = {
 	extends: [
 		'@nextcloud',
-	]
+	],
+	rules: {
+		// Allow shallow import of @vue/test-utils in order to be able to use it in
+		// the src folder
```

Comment:
Why is this needed in the src folder? :thinking:

Reply:
Ah, src/tests?
You can create another eslintrc there then, that should extend or supersed this config :wink:

Reply:
My plan was to make a PR to add it to the shared eslint config as it already contains an exception for `@vue/test-utils`.
Is it the same use case ? https://github.com/nextcloud/eslint-config/blob/master/index.js#L97

Reply:
:scream:

Reply:
https://github.com/nextcloud/eslint-config/pull/26 .........

Reply:
Not sure if that's a yes or a no :thinking:

Reply:
I did not see that pass by, I'm not sure we should have done that in the first place, but yeah :shrug: